src/main/java/duke/command/Duke.java

```diff
@@ -0,0 +1,176 @@
+                    if (value.isEmpty()) {
+                        throw new DukeException("Done Incomplete");
+                    } else {
+                        int instructNum = Integer.parseInt(value);
+                        listItem.instructionCompleted(instructNum);
+                    }
+                } else if (userCommand.contains("todo")) {
+                    String todoInstruction = userCommand.replace("todo", "").trim();
+                    if (todoInstruction.isEmpty()) {
+                        throw new DukeException("Todo Incomplete");
+                    } else {
+                        listItem.addToDo(todoInstruction);
+                    }
+                } else if (userCommand.contains("deadline")) {
+                    String deadlineInstruction = userCommand.replace("deadline", "").trim();
+                    if (deadlineInstruction.isEmpty()) {
+                        throw new DukeException("Deadline Incomplete");
+                    } else {
+                        listItem.addDeadline(deadlineInstruction);
+                    }
+                } else if (userCommand.contains("event")) {
+                    String eventInstruction = userCommand.replace("event", "").trim();
+                    if (eventInstruction.isEmpty()) {
+                        throw new DukeException("Event Incomplete");
+                    } else {
+                        listItem.addEvent(eventInstruction);
+                    }
+                } else if (userCommand.contains("help")) {
+                    helpCommands();
+                } else if (userCommand.contains("delete")) {
+                    String value = userCommand.replace("delete", "").trim();
+                    if (value.isEmpty()) {
+                        throw new DukeException("Delete Incomplete");
+                    } else {
+                        int instructNum = Integer.parseInt(value);
+                        listItem.instructionDeleted(instructNum);
+                    }
+
+                } else {
+                    throw new DukeException("Input Incomplete");
+                }
+            } catch (DukeException ignored) {
+            }
+            userCommand = sc.nextLine();
+        }
+        byeMsg();
+        try {
+            listItem.save();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void welcomeLogo() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        System.out.println("Welcome to\n" + logo);
+        displayLine();
+    }
+
+    public static void displayLine() {
+        System.out.println("----------------------------------------------------------------");
+    }
+
+    public static void greetMsg() {
```

Comment:
Do consider to follow method naming properly (should be a verb) for this and other methods as well